src/com/sun/ts/tests/signaturetest/signature-repository/javax.management.j2ee.sig_1.1_se8

```diff
@@ -23,28 +23,28 @@ CLSS public abstract interface java.io.Serializable
 
```

Comment:
Please revert all changes to this file, the *.se8 signature files will be automatically generated as described on signaturetest/README.

Reply:
Also reverted on b0ce81f